resources/views/components/partial/grades.blade.php

```diff
@@ -57,7 +57,7 @@
             <p class="partial-data"><strong>*</strong> An asterisk indicates this location did not publish enough data to evaluate. <strong>Click below to add data to the Scorecard.</strong></p>
         </div>
         <div class="right add-data">
-            <button class="btn btn-primary" onclick="document.getElementById('research').click();document.querySelector('.take-action').scrollIntoView({ behavior: 'smooth' }); return false;" {!! trackData('Nav', 'Grades', 'Add New Data') !!}>Add New Data</button>
+            <button class="btn btn-primary" onclick="window.open('https://forms.gle/WPC2Z6A92tBqxGWZ8', '_blank');" {!! trackData('Nav', 'Grades', 'Add New Data') !!}>Add New Data</button>
```

Comment:
Since this is acting as a link, vs a button ... for Accessibility purposes, we should make this an anchor tag.

```suggestion
            <a href="https://forms.gle/WPC2Z6A92tBqxGWZ8" target="_blank" class="btn btn-primary" {!! trackData('Nav', 'Grades', 'Add New Data') !!}>Add New Data</a>
```

Reply:
I agree. I also need to update the styles.

Reply:
the `btn btn-primary` should be all that is needed, but if it is not, ya, have at it ;)